card/mod/05_email/set/all/follow.rb

```diff
@@ -49,12 +49,12 @@
     when :off
       path_options['card[content]']= '[[*never]]'
       html_options[:title]         = "stop sending emails about changes to #{args[:label]}"
-      if args[:hover]
-        html_options[:hover_content] = "unfollow #{args[:label]}" 
-        html_options[:text]          = "following #{args[:label]}"
-      else
+#      if args[:hover]
+#        html_options[:hover_content] = "unfollow #{args[:label]}" 
+#        html_options[:text]          = "following #{args[:label]}"
+#      else
         html_options[:text]          = "unfollow #{args[:label]}"
-      end
+#      end
```

Comment:
the hovering is making the submenu jump around.  If that's the case, I'd rather just take out the hover behavior until we update the menu handling.